src/MenuItem/index.js

```diff
@@ -0,0 +1,31 @@
+import './styles.scss';
+
+const MenuItem = ({menuItemData}) => {
+    return (
+        <div className="col-12 col-md-4 col-lg-2">
+            <div className="card shadow mb-3 border-dark">
+                <div className="card-body">
+                    <h4 className="card-title">{menuItemData.foodName}</h4>
+                    <ul className="list-unstyled">
+                        <li className="list-group-item  rounded fw-bold"><span class='calories badge'>Calories: {menuItemData.calories}</span></li>
+                        {menuItemData.foodType &&
+                            <li className="list-group-item rounded fw-bold"><span className='type badge'>Type: {menuItemData.foodType}</span></li>
+                        }
+                        {!menuItemData.foodType &&
```

Comment:
In mobile view the calories, food type and sides fit side by side, rather than listing down the page.